### ISSUE TITLE
T1932 connection issues

### ISSUE THREAD

**Issue by SteveBarker72** [john-38787364/antifier#17]
Hi John,

Really appreciate the work you and others have put into this.
I'm hoping that the problem I have is a minor one and can be easily resolved. I can't help but think I'm missing something and I just can't see it but.....
I am using Windows 10 and I have followed all of the instructions as far as I can tell.
Antifier finds my unit OK and sees the ant dongle with problem. The issue I think I have is in receiving data from the unit. 
Zwift finds the trainer no problem but when pedalling starts, nothing happens. If i put the software into simulated trainer mode, zwift works quite merrily.
I have included a log file from a very short ride. hopefully someone can help me get to the bottom of this. It's also worth noting that when I perform a runoff, the software doesn't notice any pedalling either.
[out.log](https://github.com/john-38787364/antifier/files/1511008/out.log)

Thanks in advance. Steve

**Comment by jr111** [comment]
After starting the script, do you see your output in the Antifier window?

**Comment by SteveBarker72** [comment]
No. I've attached an image. Equally, If I try to perform a run off it just sits there asking me to ride up to 40kph.
![24271036_10204148808236528_883341054_o](https://user-images.githubusercontent.com/34070531/33472927-34842596-d66c-11e7-8fa5-cfc71abee5ff.jpg)

**Comment by jr111** [comment]
The only way I can reproduce this is when the powerback switch is off. It still finds the head unit. Your brake is verified working?
